folium/features.py

```diff
@@ -767,12 +795,20 @@ class DivIcon(MacroElement):
     def __init__(self, html=None, icon_size=None, icon_anchor=None,
                  popup_anchor=None, class_name='empty'):
         super(DivIcon, self).__init__()
-        self._name = 'DivIcon'
+        self._name = 'divIcon'
```

Comment:
Currently almost all `_name` attributes in folium are UpperCamelCase. Why should it be lower?

Reply:
Goes for all subsequent mentions; so we can interpolate each class name cleanly into its Leaflet counterpart constructor in the Jinja2 template without having to create another redundant variable name or mess with a utility to handle lowercasing the first letter. It's not an actual variable name; just a string value that gets stored. I think keeping the API as clean and straightforward as possible for other people that contribute to or maintain the project is a boon.